test/__file_snapshots__/targets--legacy-0

```diff
@@ -0,0 +1,48 @@
+// Babel targets
```

Comment:
Run `jest -u` to remove these 2 snapshots: only one now.

Reply:
Done by hand, for some reasons wasn't removed by the CLI.